.github/workflows/build-arm64-wheels.yml

```diff
@@ -59,7 +59,7 @@ jobs:
 
       - name: Install Twine
         run: |
-          if [ ! -f "venv" ]; then rm -rf venv; fi
+          if [ ! -f "venv" ]; then sudo rm -rf venv; fi
```

Comment:
why is this necessary now? Is this an unrelated regression in the build environment?